experiments/haskell-experiments/Test/Collect.py

```diff
@@ -0,0 +1,78 @@
+import argparse
+import os
+import re
+
+from benchtool.Haskell import Haskell
+from benchtool.Types import ReplaceLevel, TrialConfig, Entry
+from benchtool.Tasks import tasks
+
+
+def collect(results: str):
+    tool = Haskell(results, replace_level=ReplaceLevel.SKIP)
+
+    for workload in tool.all_workloads():
+        if workload.name not in ['LuParser']:
+            continue
+
+        with open(os.path.join(workload.path, 'app/Main.hs'), 'r+') as f:
+            oirginal_file = f.read()
+
+            old_string = r'\(allProps "src/Spec.hs"\)'
```

Comment:
Can we change this to use a more intuitive string (e.g. REPLACE ME or something to that effect)?